pulp_rpm/app/models/repository.py

```diff
@@ -51,6 +51,12 @@ class RpmRemote(Remote):
     TYPE = "rpm"
     sles_auth_token = models.CharField(max_length=512, null=True)
 
+    uln_user = models.CharField(max_length=512, null=True)
+
+    uln_pwd = models.CharField(max_length=512, null=True)
+
+    server_url = models.CharField(max_length=512, null=True)
```

Comment:
I think this field name needs to be more specific/descriptive. Something like `uln_server_url` or even `uln_server_base_url` (to make clear this is not the API authentication end point).

Reply:
Yes, I think `uln_server_base_url` is much more descriptive. Changed the name ;)